emails/views.py

```diff
@@ -135,16 +137,25 @@ def wrapped_email_test(request):
     else:
         has_attachment = True
 
-    if "has_email_tracker_study_link" in request.GET:
-        has_email_tracker_study_link = strtobool(
-            request.GET["has_email_tracker_study_link"]
+    if "has_tracker_report_link" in request.GET:
```

Comment:
Thank you for updating the wrapped email template and test view!

Reply:
Thanks for adding the test! I didn't know we added tests for this but the failed tests told me about it and was happy to add on to it!